environments/hub.json

```diff
@@ -1,5 +1,6 @@
 {
   "inputValues": {
+    "HUB_NAME": "switch",
```

Comment:
So you have the hub name in the helm PR as 'Hub', but here its `switch`. Is it ok?
Are these tests passing?